lib/ed25519.js

```diff
@@ -36,6 +45,157 @@ ed25519.constants.SEED_BYTE_LENGTH = 32;
 ed25519.constants.SIGN_BYTE_LENGTH = 64;
 ed25519.constants.HASH_BYTE_LENGTH = 64;
 
+/**
+ * ASN1 public key validator for importing ed25519 keys from node >= 12
```

Comment:
```suggestion
 * ASN1 public key validator for importing ed25519 keys from Node.js >= 12
```

Reply:
Also change "node" elsewhere in this PR to "Node.js" for clarity.

Reply:
we're actually removing node in favor of DER formatted so other crypto libraries are included.